### DISCUSSION TITLE
Security Going forward...

### DISCUSSION THREAD

**Discussion by hs6666** [Ylianst/MeshCentral#5075] [Ideas]
Hi , now that (understandably), this software is no longer actively developed, thank you Ylianst for all your work on this project all this time :)

I was looking at the architecture overview and noticed the npm dependencies used ....

"archiver": "^3.0.0", Handles ZIP archives
"body-parser": "^1.18.2", Handles HTTP form posts
"compression": "^1.7.3", Handles ZIP archives
"connect-redis": "^3.4.0", HTTP session store
"cookie-session": "^2.0.0-beta.3", Cookie based HTTP sessions
"express": "^4.16.4", Handles HTTP requests
"express-handlebars": "^3.0.0", Modifies HTML response documents
"express-session": "^1.15.6", Handles HTTP sessions
"express-ws": "^4.0.0", Handles WebSocket
"minimist": "^1.2.0", Parses input arguments
"multiparty": "^4.2.1", Handles HTTP form posts
"nedb": "^1.8.0", Light MongoDB substitute
"node-forge": "^0.7.6", Encryption library
"util.promisify": "^1.0.0", Poly-fill Node 6/7
"ws": "^6.1.2", Web socket client
"xmldom": "^0.1.27", XML parsing
"yauzl": "^2.10.0" Handles ZIP archives 

Considering the "power" of meshcentral, making it a juicy target.... was wondering if anyone knows of a site were you can add the above list, so as to to be notified of any future security vulnerabilities in these npm packages

I for one would be happy to donate towards any third party services that offer these kind of services

Cheers

Harry

**Comment by si458** [comment]
i use newreleases.io
login, then link github account/npm account, then use the github repo of the project or npm package
i get emails when new releases of software happen
(i juse it for meshcentral hehe)
